expression/builtin_string_vec.go

```diff
@@ -62,6 +62,31 @@ func (b *builtinRepeatSig) vecEvalString(input *chunk.Chunk, result *chunk.Colum
 	return nil
 }
 
+func (b *builtinStringIsNullSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
+	n := input.NumRows()
+	buf, err := b.bufAllocator.get(types.ETString, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(buf)
+	if err := b.args[0].VecEvalString(b.ctx, input, buf); err != nil {
+		return err
+	}
+
+	for i := 0; i < n; i++ {
+		if buf.IsNull(i) {
+			result.AppendInt64(1)
```

Comment:
Before you use the "result", you should resize the "result". Besides, you can call `result.Int64s()` function to get the golang slice for calculating fast.

Reply:
ok

Reply:
Here is [an example](https://github.com/pingcap/tidb/pull/12018/files#diff-92183526d2de28ba557154b007de3e41R33) of how to use `ResizeInt64` + `Int64s` to modify a column, PTAL ~